src/blocks/IconForestBlock.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import {
+  ContentWrapper,
+  SectionWrapper,
+  TABLET_BP_DOWN,
+} from '../components/blockHelpers'
+import { BaseBlockProps, MarkdownHtmlComponent } from './BaseBlockProps'
+
+const ColumnContentWrapper = styled(ContentWrapper)({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  textAlign: 'center',
+})
+
+const Title = styled('h2')({
+  fontSize: '3.75rem',
+  width: '100%',
+  margin: 0,
+  [TABLET_BP_DOWN]: {
+    fontSize: '2.25rem',
+  },
+})
+
+const Picture = styled('picture')({
+  maxWidth: '47.125rem',
```

Comment:
i don't think it matters much if this is `47` or a`47.125` rems does it? I think we should avoid these "magic" decimals if we can

Reply:
I agree :) I'll stick to integers in the future.

Reply:
totally agree!